internal/config/profile.go

```diff
@@ -271,8 +271,10 @@ func (p *Profile) IsAccessSet() bool {
 func Map() map[string]string { return p.Map() }
 func (p *Profile) Map() map[string]string {
 	settings := viper.GetStringMapString(p.Name())
-	newSettings := make(map[string]string, len(settings))
-
+	newSettings := make(map[string]string, len(settings)+1)
+	if p.MongoShellPath() != "" {
+		newSettings[mongoShellPath] = p.MongoShellPath()
+	}
```

Comment:
small drive by, we set the shell path as a global things instead of per profile and I wasn't sure why it wasn't showing when doing describe